applications/FluidDynamicsApplication/custom_utilities/drag_utilities.cpp

```diff
@@ -118,6 +118,90 @@ namespace Kratos
         return drag_force_center;
     }
 
+
+    array_1d<double, 3> DragUtilities::CalculateBodyFittedBaseMoment(ModelPart& rModelPart, array_1d<double, 3> rReferencePoint) {
```

Comment:
Isn't this function the same of `base_moment_utilities.cpp`?

Reply:
@rubenzorrilla Is it ok to include this functionality in drag_utilities.cpp or a separate base_moment_utilities is required ?

Reply:
As I discussed with @riccardotosi in the office, in here I see two options:
1. If it has to be a separated `base_moment_utilities` file, I'd definitively move it to the `ExaquteSandboxApplication` so we get rid of maintaining it once the project finishes.
2. Considering that this utility may eventually be used for other purposes (i.e. calculating the pitching moment) we could think about including it in the `FluidDynamicsApplication`.

Taking into account that the `FluidDynamicsApplication` is supposed to be a core application, for option 2 we would need to:
- Use the `parallel_utils`
- Follow the style guide standards (removing commented code, indentation, ...)
- Change the name of the utility and methods to remove the `Base...` (this is misleading as the moment is computed with respect to a user-defined point).
- Enhance testing. If I'm not wrong, right now we check an in-plane drag force with (0,0,0) as reference point. This might eventually lead to false positiveness of the test.
- We should definitively not include 3 extra Python scripts to the application. For this we need to discuss about how do we want to output the drag and the moment. Most probably we want to merge everything in one process.

Taking everything into account, my personal view is to include it as it is right now in the application that was specifically created for the project so we don't interfere with the project. In here I'd like to note that the `FluidDynamicsApplication` is a hard to maintain application which is going through a (slow) clean up process. At some point we may think about rescuing it from the project app.